lab_1_classify_by_unigrams/start.py

```diff
@@ -1,22 +1,31 @@
 """
 Language detection starter
 """
-# pylint:disable=too-many-locals, unused-argument, unused-variable
+
```

Comment:
remove these empy lines and add a new one after `pylint disable`

Reply:
done, it seems